packages/storybook-angular/src/lib/preset.ts

```diff
@@ -141,9 +141,27 @@ function angularOptionsPlugin(options, { normalizePath }) {
           imports.push('zone.js');
         }
 
+        const projectConfig =
+          await options.angularBuilderContext.getProjectMetadata(
+            options.angularBuilderContext.target.project,
+          );
+
         return {
           code: `
-            ${imports.map((extraImport) => `import '${extraImport}';`).join('\n')}
+            ${imports
+              .map((extraImport) => {
+                if (
+                  extraImport.startsWith('.') ||
+                  extraImport.startsWith('src')
+                ) {
+                  // relative to root
+                  return `import '${resolve(options.angularBuilderContext.workspaceRoot, projectConfig.root, extraImport)}';`;
+                }
+
+                // absolute import
+                return `import '${extraImport}';`;
+              })
```

Comment:
Hey @brandonroberts, actually there is no need in extra conditions here as `path.resolve('/abosulte_path_to_project', '/absolute_path_to_style')` will resolve to `/absolute_path_to_style` so all this code code can be replace with a simple return `import '${resolve(options.angularBuilderContext.workspaceRoot, projectConfig.root, extraImport)}';` and it will cover all the cases.

Reply:
For example in our project all the code is located not in the `src` directory, so specifying `styles: ['path/to/our/src/global.scss']` still won't work. I understand that we could start the path with a dot, but why introduce these confusing rules if they can we avoided. 🤷‍♂️